.ci/install_crio.sh

```diff
@@ -12,11 +12,14 @@ source "${cidir}/lib.sh"
 
 echo "Get CRI-O sources"
 crio_repo="github.com/kubernetes-incubator/cri-o"
-crio_version=$(get_version "externals.crio.version")
 go get -d "$crio_repo" || true
 pushd "${GOPATH}/src/${crio_repo}"
-git fetch
-git checkout "${crio_version}"
+
+if [ "$ghprbGhRepository" != "${crio_repo/github.com\/}" ];
+	crio_version=$(get_version "externals.crio.version")
+	git fetch
+	git checkout "${crio_version}"
```

Comment:
We need to test a specific PR, we're actually not testing master, and we're not testing a specific version either.

Reply:
@sboeuf that is right, but I am checking-out to the specific PR on the jenkins job configuration. Which does this:
```
crio_repo="github.com/kubernetes-incubator/cri-o"
tests_repo="github.com/kata-containers/tests"
go get -d "$crio_repo" || true
go get -d "$tests_repo" || true

cd "${GOPATH}/src/${crio_repo}"

git fetch origin "pull/${pr_number}/head:${pr_branch}"
git checkout "${pr_branch}"
git rebase "origin/${ghprbTargetBranch}"
```

And this line only checkout to the `$crio_version` on our `versions.yaml` only if the the PR does not come from the `kubernetes-incubator/cri-o` repository.

Reply:
Yep definitely, I have read the code in the wrong direction, sorry...